pagopa/pn-frontend-e2e
# Pn 9248 e 2 e mittente invio notifica digitale a destinatario con domicilio speciale impostato caso ko al secondo tentativo invio raccomandata semplice

## Short description

Created new tests: 9244 - 9245 - 9246 - 9247 - 9248

## List of changes proposed in this pull request

- Created new tests: PA sends a notification to different type of dimicile and verify the failure or correct sending
- All tests are in the same file. To launch the suite use the tag @invioNotificaDigitaleADomicilioImpostato
- Updated methods to manage the contact details in background

## How to test

Simply run `mvn clean test -D"cucumber.filter.tags=@invioNotificaDigitaleADomicilioImpostato"` and verify if the tests works fine.
